Protocol.md

```diff
@@ -66,6 +66,7 @@ the funding transaction and CETs.
 
 1. type: 42778 (`offer_dlc_v0`)
 2. data:
+   * [`bigsize`: `protocol_version`]
```

Comment:
I believe `u16`:`num_funding_inputs` is a `bigsize` in the test vectors (specifically `enum_single_oracle_test.json`)

Reply:
Thanks I had forgotten to update it, fixed.